src/cart/cart.controller.ts

```diff
@@ -0,0 +1,74 @@
+import express from "express";
+import { Response, Request } from "express-serve-static-core";
```

Comment:
I guess you could import it in that way:

```suggestion
import express, { Response, Request } from "express";
```

Reply:
fixed